crates/holochain_zome_types/src/lib.rs

```diff
@@ -70,6 +70,7 @@ pub mod test_utils;
 pub use entry::Entry;
 pub use header::Header;
 pub use prelude::*;
+pub use subtle;
```

Comment:
I like to distinguish re-exported dependencies:

```suggestion
/// Re-exported dependencies
pub mod dependencies {
    pub use ::subtle;
}
```